isis/appdata/serialnumbers/Apollo15SerialNumber.trn

```diff
@@ -0,0 +1,30 @@
+Group = Keyword1
```

Comment:
Is this file used for a specific instrument?

Reply:
Not sure what this one is used for. There are already serial number files for metric and panoramic camera so I'm guessing that this one is not specific to either